bin/install-wp-tests.sh

```diff
@@ -150,6 +150,17 @@ install_db() {
 	mysqladmin create $DB_NAME --user="$DB_USER" --password="$DB_PASS"$EXTRA
```

Comment:
We need to drop the database first, otherwise the script fails if the database is present
```suggestion
	mysqladmin drop $DB_NAME -f --user="$DB_USER" --password="$DB_PASS"$EXTRA
	mysqladmin create $DB_NAME --user="$DB_USER" --password="$DB_PASS"$EXTRA
```